tests/phpunit/tests/fonts/font-library/wpFontsDir.php

```diff
@@ -69,4 +81,57 @@ function set_new_values( $defaults ) {
 
 		$this->assertSame( static::$dir_defaults, $font_dir, 'The wp_get_font_dir() method should return the default values.' );
 	}
+
+	public function test_should_created_fonts_dir_in_uploads_when_fails_in_wp_content() {
```

Comment:
```suggestion
	public function test_should_create_fonts_dir_in_uploads_when_fails_in_wp_content() {
```
Fixes small typo.

Would `test_should_create_fonts_dir_in_uploads_when_wp_content_not_writable` be clearer here?